src/ibrowse.erl

```diff
@@ -335,6 +330,20 @@ send_req(Url, Headers, Method, Body, Options, Timeout) ->
             {error, {url_parsing_failed, Err}}
     end.
 
+lb_pid(Host, Port, Url) ->
+    case ets:lookup(ibrowse_lb, {Host, Port}) of
+        [] ->
+            get_lb_pid(Url);
+        [#lb_pid{pid = Lb_pid_1}] ->
```

Comment:
Minor nit: don't need the Lb_pid_1 as there is only one Pid in this function

Reply:
Ah, good point! I just copied that code block without thinking about simplification. I have updated the PR renaming `Lb_pid_1` to simply `Pid`. Thanks for the eagle eye. I'll update the upstream PR as well.